index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Sputnik Realty</title>
+	<link rel="stylesheet" href="main.css">
+</head>
+<body>
+	<header id="topArea">
+		<a href="index.html" id="logo">
+			<h1>Sputnik Realty</h1>
+			<h3>Get your place in Outer Space!</h3>
+		</a>
+		<nav>
+      <div id="topNav">
```

Comment:
note that the 'nav' semantic tag is technically a div- you might not need this one since Joe can select the nav bar using the nav tag. up to you guys though.